go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/submariner-io/lighthouse v0.17.2
 	github.com/submariner-io/shipyard v0.17.2
 	github.com/submariner-io/submariner v0.17.2
-	github.com/submariner-io/submariner-operator v0.17.2
+	github.com/submariner-io/submariner-operator v0.17.3-0.20240701075427-c468b6f7b1df
```

Comment:
This will need to be updated when https://github.com/submariner-io/submariner-operator/pull/3143 is merged.